.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "editor.codeActionsOnSave": {
-        "source.organizeImports": true
+      "source.organizeImports": "explicit"
```

Comment:
What does this do?

Reply:
VS Code update no longer uses boolean values: https://stackoverflow.com/a/77637765/1935442